neurons/validator.py

```diff
@@ -35,7 +35,7 @@
 # import base validator class which takes care of most of the boilerplate
 from folding.store import Job
 from folding.base.validator import BaseValidatorNeuron
-from folding.utils.logging import log_event
+from folding.utils.folding_log import log_event
```

Comment:
this wont work because it's a method of the class

Reply:
https://github.com/macrocosm-os/folding/pull/147/commits/7bc216ec6bb04da25fc913749ad4c8db80007625